libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -488,32 +488,37 @@ method publish*(g: GossipSub,
 
   var peers: HashSet[PubSubPeer]
 
+  # add always direct peers
+  peers.incl(g.explicit.getOrDefault(topic))
+
+  if topic in g.topics: # if we're subscribed use the mesh
+    peers.incl(g.mesh.getOrDefault(topic))
+
   if g.parameters.floodPublish:
+    let
+      msgSize = data.len
+      bandwidth = 25_000_000 #TODO replace with bandwidth estimate
```

Comment:
I suspect we need to make this a (debug) parameter until we have something better - ie in nimbus-eth2, this would be a hidden command-line parameter that we pull out in case of weird behavior - hidden, because obviously it's something that should go away at some point